contributors/devel/development.md

```diff
@@ -279,20 +279,42 @@ To build binaries for all platforms:
 make cross
 ```
 
+#### Install etcd
+```
+hack/install-etcd.sh
+```
+
+#### Verification
+```
+# Run all hack/verify-*.sh scripts
+make verify
+```
+
 #### Test
 
 ```sh
 cd $working_dir/kubernetes
 
+# Run all hack/update-*.sh scripts
+# For small change, run "make verify" and then relevant scripts, for example:
+# hack/update-gofmt.sh (when adding new files)
+# hack/update-bazel.sh (when updating dependencies / imports)
+make update
+
 # Run every unit test
 make test
 
 # Run package tests verbosely
 make test WHAT=pkg/util/cache GOFLAGS=-v
+# Or use bazel (quote "..." when using zsh)
+bazel test //pkg/util/cache/...
```

Comment:
Not sure if we need to add this, because everywhere else doesn't use bazel to build or run tests. 
We did mention bazel and said it's experimental and linked to its doc https://github.com/kubernetes/community/blob/master/contributors/devel/bazel.md

Therefore, I'd suggest we just remove this for now.